iceddev/git-splainin
# Images

#### What's this PR do?

Adds good quality icons and fixes permissions to use rawgit.
#### Where should the reviewer start?

manifest.js has most of the image and permission changes.
#### How should this be manually tested?

Check out the images in the extension page, options tab and PR page. Try to add a template url from rawgit.
#### Any background context you want to provide?

N/A
#### What are the relevant tickets?

closes #17
